README.md

```diff
@@ -78,7 +86,13 @@ module.exports = {
         }
       },
       {
-        enforce: 'post', // required
+        // Loaders that compile css (like postcss, less or sass) should be left as normal loaders
+        test: /\.css$/,
+        use: [ 'e.g. postcss-loader/less-loader/sass-loader' ]
```

Comment:
Oops, I forgot about the extension. I agree that we should have a full working example that you suggested!
One thing I would like to suggest is that we should use `.scss` extension if we use `sass-loader` without any options, as far as I know.